README.md

```diff
@@ -40,24 +44,29 @@ The playground runs a number of services. The TCP ports used may clash with exis
 | playground-trino      | 18080                |
 | playground-jupyter    | 18888                |
 
-## Start playground
+## Playground usage
+
+### Launch playground
 
-### Launch all components of playground
 ```shell
 git clone git@github.com:apache/gravitino-playground.git
 cd gravitino-playground
-./launch-playground.sh
+./playground.sh start
 ```
+> **Note:** If you install Docker with sudo in Linux OS, please run playground.sh with sudo to ensure it can access all necessary information as root user.
 
-### Launch special component or components of playground
 ```shell
-git clone git@github.com:apache/gravitino-playground.git
-cd gravitino-playground
-./launch-playground.sh hive|gravitino|trino|postgresql|mysql|spark|jupyter
+sudo ./playground.sh start
```

Comment:
Why do we need `sudo`?

Reply:
Install Docker should run command with sudo in Linux OS.  If user didn't add current login user(non root) to the Docker group,  he need to use docker command with sudo. 
Another reason is that I use lsof -i : to check the port, lsof only get the current login user process while the docker process is in root user

Reply:
`sudo` will be a burden to use this script.

Reply:
<img width="748" alt="image" src="https://github.com/user-attachments/assets/7bb48ccd-6e9c-4745-9104-e4c208b6db76">

Reply:
Updated script and doc. Add `sudo lsof` only for ports check in Linux